packages/ensnode-sdk/src/ensindexer/config/zod-schemas.test.ts

```diff
@@ -45,8 +49,8 @@ describe("ENSIndexer: Config", () => {
           ),
         ).toContain("Plugins cannot contain duplicate values");
 
-        expect(formatParseError(makePluginsListSchema().safeParse([]))).toContain(
-          "Plugins must be a list with at least one valid plugin name. Valid plugins are: subgraph, basenames, lineanames, threedns, reverse-resolvers, referrals, tokenscope",
+        expect(formatParseError(makePluginsListSchema().safeParse([]))).toMatch(
```

Comment:
made this error message check more general so the test is less fragile but still accomplishes its goal